src/pkg/selectors/testdata/details.go

```diff
@@ -323,3 +325,21 @@ func GetDetailsSet() *details.Details {
 		},
 	}
 }
+
+func MakeErrors(failure, recovered, skipped bool) fault.Errors {
+	fe := fault.Errors{}
+
+	if failure {
+		fe.Failure = assert.AnError
+	}
+
+	if recovered {
+		fe.Recovered = []error{assert.AnError}
```

Comment:
nit: can you wrap this or something so it's not the exact same as the failure case